Formula/carla.rb

```diff
@@ -35,7 +35,7 @@ def install
     system "make", "install", "PREFIX=#{prefix}"
 
     inreplace bin/"carla", "PYTHON=$(which python3 2>/dev/null)",
-                           "PYTHON=#{Formula["python@3.10"].opt_bin}/python3.10"
+                           "PYTHON=#{Formula["python@3.11"].opt_bin}/python3.11"
```

Comment:
There seem to be a number of scripts with same logic:
```console
❯ rg -F 'PYTHON=$(which python3 2>/dev/null)' carla/2.5.2/bin -l | wc -l
      17
```

Since `python@3.10` still owns `python3` command, those scripts may not work correctly if only `python@3.11` is installed.